src/locales/fr.ts

```diff
@@ -202,7 +202,7 @@ export default {
     eusebius_of_vercelli_bishop: 'Saint Eusèbe de Verceil, Evêque (✝ 371)',
     fabian_i_pope: 'Saint Fabien, Pape et Martyr (✝ 250)',
     fidelis_of_sigmaringen_priest: 'Saint Fidèle de Sigmaringen, Prêtre et Martyr (✝ 1622)',
-    first_martyrs_of_the_holy_roman_church: 'Premiers Saints martyrs de la Sainte Église romaine (✝ 64)',
+    first_martyrs_of_the_holy_roman_church: 'Premiers martyrs de l''Église de Rome (✝ 64)',
```

Comment:
OK. I'm going to try...

Reply:
Here ! And now I know why I see oblique quotes so often in French translations of English texts when we don't have the use of them;)

Reply:
Yes... not in javascript. Thanks for the fix!

Reply:
> Here ! And now I know why I see oblique quotes so often in French translations of English texts when we don't have the use of them;)
Sorry, I was sure a double quote would escape the character ...

@Scriptura, well, while that might be a partial reason to the usage of [typographical apostrophe](https://en.m.wikipedia.org/wiki/Apostrophe#Typographic_form) (you call the oblique apostrophe), my personal reason is that I prefer it over the typewriter apostrophe as it looks better. Some might argue that the client application should change the apostrophe form rather than hard-code it within the code.